portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,190 @@
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+  margin-left: 15%;
+  margin-right: 15%;
 }
 
 #greeting-container {
   margin-top: 20px;
+  
+}
+
+#TechCorner {
+    background-color: #F0EDE5;
+    text-align: right;
+}
+
+#Contact {
+    background-color: #F0EDE5;
+    text-align: right;
+}
+
+
+img.image {
+    float: right;
+    padding: 20px;
+}
+
+p.intro, div a{
+    width: 50%;
+    white-space: pre-wrap;  
+    word-wrap: break-word;
+
+}
+
+/* travel log slideshow setting */
+.slideshow-container{
+    box-sizing: border-box;
+    width: 50%;
+    position: relative;
+    margin: auto;
+}
+
+
+#firstSlide {
+    box-sizing: border-box;
+    display: block;
+}
+
+.mySlides {
+    box-sizing: border-box;
+    display: none;
+}
+
+
+/* Next & previous buttons */
+.prev, .next {
+  cursor: pointer;
+  position: absolute;
+  top: 50%;
+  width: auto;
+  margin-top: -22px;
+  padding: 16px;
+  color: white;
+  font-weight: bold;
+  font-size: 18px;
+  transition: 0.6s ease;
+  border-radius: 0 3px 3px 0;
+  user-select: none;
+}
+
+/* Position the "next button" to the right */
+.next {
+  right: 0;
+  border-radius: 3px 0 0 3px;
+}
+
+/* On hover, add a black background color with a little bit see-through */
+.prev:hover, .next:hover {
+  background-color: rgba(0,0,0,0.8);
+}
+
+/* Caption text */
+.text {
+  color: #EAE6DA;
+  font-size: 20px;
+  font-family: "Lato";
+  padding: 8px 12px;
+  position: absolute;
+  bottom: 8px;
+  width: 100%;
+  text-align: center;
+  text-shadow: 2px 2px #2E4A62;
+}
+
+/* Number text (1/3 etc) */
+.numbertext {
+  color: #EAE6DA;
+  font-size: 18px;
+  font-family: "Lato";
+  padding: 8px 12px;
+  position: absolute;
+  top: 0;
+}
+
+/* Fading animation */
+.fade {
+  -webkit-animation-name: fade;
+  -webkit-animation-duration: 1.5s;
+  animation-name: fade;
+  animation-duration: 1.5s;
+}
+
+@-webkit-keyframes fade {
+  from {opacity: .4} 
+  to {opacity: 1}
+}
+
+@keyframes fade {
+  from {opacity: .4} 
+  to {opacity: 1}
+}
+/* travel log slideshow setting */
+
+
+/* general setting */
+h1 {
+    color: #C0AB8E;
+    text-align: center;
+    font-family: "Major Mono Display";
+    font-size: 45px;
+}
+
+h2 {
+    color: #A9754F;
+    font-family: "Nothing You Could Do";
+    font-size: 35px;
+}
+
+p, a {
+    font-family: "Lato";
+    color: #2E4A62;
+    font-size: 25px;
+    text-decoration: none;
+    padding: 18px;
 }
+
+a:hover {
+    text-decoration: underline;
+}
+
+/* general setting */
+
+
+
+/* navigation bar setting */
+ul {
+    list-style-type: none;
+    overflow: hidden;
+    margin: 0;
+    padding: 0;
+    text-align: center;
+    position: sticky;
+    top: 0;
+    z-index: 100;
+}
+
+li {
+    float: left;
+    width: 25%;
+}
+
+li a {
+  font-family: "Codystar";
+  font-size: 23px;
+  display: block;
+  color: #485167;
+  text-decoration: none;
+  text-align: center;
+  padding: 14px 16px;
+} 
+
+li a:hover {
+    color: #B4B7BA;
+}
+/* navigation bar setting */
```

Comment:
It is possible that you would later add some other lists to your page. This styling is specific to the navbar so it would be better to specifically target that. Example: `.navbar` and `.navbar li`

Reply:
done:)